server/build.gradle.kts

```diff
@@ -35,6 +42,8 @@ dependencies {
 	implementation("org.jacoco:org.jacoco.core:$jacocoVersion")
 	implementation(platform("com.google.cloud:libraries-bom:24.1.0"))
 	implementation("com.google.cloud:google-cloud-storage")
+	implementation( "com.google.android.gms:play-services-vision:19.0.0")
+	implementation("com.google.android.gms:play-services-mlkit-text-recognition:$textRecognitionVersion")
```

Comment:
Why do we have text recognition in our server dependencies?

Reply:
Dependencies deleted